Traceback (most recent call last):
  File "./main_bytes.py", line 40, in <module>
    arr = load_file(fname)
  File "/home/divesh/Desktop/thesis/offline/load_file.py", line 29, in load_file
    cap.apply_on_packets(new_packet)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/capture/capture.py", line 277, in apply_on_packets
    return self.eventloop.run_until_complete(coro)
  File "/usr/lib/python3.6/asyncio/base_events.py", line 471, in run_until_complete
    self.run_forever()
  File "/usr/lib/python3.6/asyncio/base_events.py", line 438, in run_forever
    self._run_once()
  File "/usr/lib/python3.6/asyncio/base_events.py", line 1415, in _run_once
    event_list = self._selector.select(timeout)
  File "/usr/lib/python3.6/selectors.py", line 445, in select
    fd_event_list = self._epoll.poll(timeout, max_ev)
KeyboardInterrupt
Traceback (most recent call last):
  File "./main_bytes.py", line 22, in <module>
    error_file = open(error_path,"a")
FileNotFoundError: [Errno 2] No such file or directory: '../output/choosen_traffic/error_momon_uti_with_param.csv'
Traceback (most recent call last):
  File "./main_bytes.py", line 22, in <module>
    error_file = open(error_path,"a")
FileNotFoundError: [Errno 2] No such file or directory: '../output/choosen_traffic/error_momon_uti_with_param.csv'
Traceback (most recent call last):
  File "./main_bytes.py", line 22, in <module>
    error_file = open(error_path,"a")
FileNotFoundError: [Errno 2] No such file or directory: '../output/choosen_traffic/error_momon_uti_with_param.csv'
Traceback (most recent call last):
  File "./main_bytes.py", line 22, in <module>
    error_file = open(error_path,"a")
FileNotFoundError: [Errno 2] No such file or directory: '../output/choosen_traffic/error_momon_uti_with_param.csv'
Traceback (most recent call last):
  File "./main_bytes.py", line 22, in <module>
    error_file = open(error_path,"a")
FileNotFoundError: [Errno 2] No such file or directory: '../output/choosen_traffic/error_momon_uti_with_param.csv'
Traceback (most recent call last):
  File "./main_bytes.py", line 22, in <module>
    error_file = open(error_path,"a")
FileNotFoundError: [Errno 2] No such file or directory: '../output/choosen_traffic/error_momon_uti_with_param.csv'
Traceback (most recent call last):
  File "./main_bytes.py", line 6, in <module>
    import matplotlib.pyplot as plt
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/matplotlib/pyplot.py", line 32, in <module>
    import matplotlib.colorbar
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/matplotlib/colorbar.py", line 31, in <module>
    import matplotlib.contour as contour
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/matplotlib/contour.py", line 16, in <module>
    import matplotlib.font_manager as font_manager
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/matplotlib/font_manager.py", line 1341, in <module>
    fontManager = json_load(_fmcache)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/matplotlib/font_manager.py", line 937, in json_load
    return json.load(fh, object_hook=_json_decode)
  File "/usr/lib/python3.6/json/__init__.py", line 299, in load
    parse_constant=parse_constant, object_pairs_hook=object_pairs_hook, **kw)
  File "/usr/lib/python3.6/json/__init__.py", line 367, in loads
    return cls(**kw).decode(s)
  File "/usr/lib/python3.6/json/decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python3.6/json/decoder.py", line 355, in raw_decode
    obj, end = self.scan_once(s, idx)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/matplotlib/font_manager.py", line 890, in _json_decode
    cls = o.pop('__class__', None)
KeyboardInterrupt
Traceback (most recent call last):
  File "./main_bytes.py", line 11, in <module>
    from similarity import *
  File "/home/divesh/Desktop/thesis/offline/similarity.py", line 1, in <module>
    import similaritymeasures
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/similaritymeasures/__init__.py", line 6, in <module>
    __version__ = open(VERSION_FILE).read().strip()
  File "/usr/lib/python3.6/codecs.py", line 309, in __init__
    IncrementalDecoder.__init__(self, errors)
KeyboardInterrupt
Traceback (most recent call last):
  File "./main_bytes.py", line 4, in <module>
    import numpy
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/__init__.py", line 142, in <module>
    from . import core
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/core/__init__.py", line 82, in <module>
    from . import defchararray as char
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/core/defchararray.py", line 754, in <module>
    @array_function_dispatch(_unary_op_dispatcher)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/core/overrides.py", line 188, in decorator
    exec(source_object, scope)
  File "<__array_function__ internals>", line 2, in <module>
  File "/usr/lib/python3.6/functools.py", line 86, in wraps
    assigned=assigned, updated=updated)
KeyboardInterrupt
Traceback (most recent call last):
  File "./main_bytes.py", line 4, in <module>
    import numpy
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/__init__.py", line 145, in <module>
    from . import lib
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/lib/__init__.py", line 30, in <module>
    from .shape_base import *
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 951, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 894, in _find_spec
  File "<frozen importlib._bootstrap_external>", line 1157, in find_spec
  File "<frozen importlib._bootstrap_external>", line 1129, in _get_spec
  File "<frozen importlib._bootstrap_external>", line 1271, in find_spec
  File "<frozen importlib._bootstrap_external>", line 96, in _path_isfile
  File "<frozen importlib._bootstrap_external>", line 88, in _path_is_mode_type
  File "<frozen importlib._bootstrap_external>", line 82, in _path_stat
KeyboardInterrupt
Traceback (most recent call last):
  File "./main_bytes.py", line 22, in <module>
    error_file = open(error_path,"a")
FileNotFoundError: [Errno 2] No such file or directory: '../output/choosen_traffic/error_momon_uti_with_param.csv'
Traceback (most recent call last):
  File "./main_bytes.py", line 22, in <module>
    error_file = open(error_path,"a")
FileNotFoundError: [Errno 2] No such file or directory: '../output/choosen_traffics/error_curv_momon_uti_with_param.csv'
Traceback (most recent call last):
  File "./main_bytes.py", line 22, in <module>
    error_file = open(error_path,"a")
FileNotFoundError: [Errno 2] No such file or directory: '../output/choosen_traffics/error_curv_momon_uti_with_param.csv'
Traceback (most recent call last):
  File "./main_bytes.py", line 22, in <module>
    error_file = open(error_path,"a")
FileNotFoundError: [Errno 2] No such file or directory: '../output/choosen_traffics/error_curv_momon_uti_with_param.csv'
Traceback (most recent call last):
  File "./main_bytes.py", line 22, in <module>
    error_file = open(error_path,"a")
FileNotFoundError: [Errno 2] No such file or directory: '../output/choosen_traffics/error_curv_momon_uti_with_param.csv'
Traceback (most recent call last):
  File "./main_bytes.py", line 4, in <module>
    import numpy
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/__init__.py", line 142, in <module>
    from . import core
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/core/__init__.py", line 82, in <module>
    from . import defchararray as char
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/core/defchararray.py", line 912, in <module>
    @array_function_dispatch(_unary_op_dispatcher)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/core/overrides.py", line 181, in decorator
    source, filename='<__array_function__ internals>', mode='exec')
KeyboardInterrupt
Traceback (most recent call last):
  File "./main_bytes.py", line 22, in <module>
    error_file = open(error_path,"a")
FileNotFoundError: [Errno 2] No such file or directory: '../output/choosen_traffics/error_curv_momon_uti_with_param.csv'
Traceback (most recent call last):
  File "./main_bytes.py", line 22, in <module>
    error_file = open(error_path,"a")
FileNotFoundError: [Errno 2] No such file or directory: '../output/choosen_traffics/error_curv_momon_uti_with_param.csv'
Traceback (most recent call last):
  File "./main_bytes.py", line 34, in <module>
    arr = load_file(fname)
  File "/home/divesh/Desktop/thesis/offline/load_file.py", line 32, in load_file
    cap.apply_on_packets(new_packet)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/capture/capture.py", line 277, in apply_on_packets
    return self.eventloop.run_until_complete(coro)
  File "/usr/lib/python3.6/asyncio/base_events.py", line 471, in run_until_complete
    self.run_forever()
  File "/usr/lib/python3.6/asyncio/base_events.py", line 438, in run_forever
    self._run_once()
  File "/usr/lib/python3.6/asyncio/base_events.py", line 1451, in _run_once
    handle._run()
  File "/usr/lib/python3.6/asyncio/events.py", line 145, in _run
    self._callback(*self._args)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/capture/capture.py", line 288, in packets_from_tshark
    await self._go_through_packets_from_fd(tshark_process.stdout, packet_callback, packet_count=packet_count)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/capture/capture.py", line 306, in _go_through_packets_from_fd
    psml_structure=psml_struct)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/capture/capture.py", line 363, in _get_packet_from_stream
    packet = packet_from_xml_packet(packet, psml_structure=psml_structure)
KeyboardInterrupt
Failed to import the site module
Traceback (most recent call last):
  File "/usr/lib/python3.6/site.py", line 570, in <module>
    main()
  File "/usr/lib/python3.6/site.py", line 553, in main
    known_paths = venv(known_paths)
  File "/usr/lib/python3.6/site.py", line 490, in venv
    addsitepackages(known_paths, [sys.prefix])
  File "/usr/lib/python3.6/site.py", line 345, in addsitepackages
    addsitedir(sitedir, known_paths)
  File "/usr/lib/python3.6/site.py", line 213, in addsitedir
    addpackage(sitedir, name, known_paths)
  File "/usr/lib/python3.6/site.py", line 174, in addpackage
    exec(line)
  File "<string>", line 1, in <module>
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/_virtualenv.py", line 42, in <module>
    from threading import Lock
  File "/usr/lib/python3.6/threading.py", line 7, in <module>
    from traceback import format_exc as _format_exc
  File "/usr/lib/python3.6/traceback.py", line 5, in <module>
    import linecache
  File "/usr/lib/python3.6/linecache.py", line 11, in <module>
    import tokenize
  File "/usr/lib/python3.6/tokenize.py", line 33, in <module>
    import re
  File "/usr/lib/python3.6/re.py", line 123, in <module>
    import sre_compile
  File "/usr/lib/python3.6/sre_compile.py", line 14, in <module>
    import sre_parse
  File "/usr/lib/python3.6/sre_parse.py", line 15, in <module>
    from sre_constants import *
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 674, in exec_module
  File "<frozen importlib._bootstrap_external>", line 779, in get_code
  File "<frozen importlib._bootstrap_external>", line 487, in _compile_bytecode
KeyboardInterrupt
Traceback (most recent call last):
  File "./main_bytes.py", line 4, in <module>
    import numpy
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/__init__.py", line 145, in <module>
    from . import lib
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/lib/__init__.py", line 38, in <module>
    from .arraysetops import *
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/lib/arraysetops.py", line 153, in <module>
    return_counts=False, axis=None):
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/core/overrides.py", line 168, in decorator
    verify_matching_signatures(implementation, dispatcher)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/core/overrides.py", line 89, in verify_matching_signatures
    if dispatcher_spec.defaults != (None,) * len(dispatcher_spec.defaults):
KeyboardInterrupt
Traceback (most recent call last):
  File "./main_bytes.py", line 4, in <module>
    import numpy
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/__init__.py", line 145, in <module>
    from . import lib
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/lib/__init__.py", line 27, in <module>
    from .index_tricks import *
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/lib/index_tricks.py", line 14, in <module>
    from .function_base import diff
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/lib/function_base.py", line 4641, in <module>
    def append(arr, values, axis=None):
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/core/overrides.py", line 181, in decorator
    source, filename='<__array_function__ internals>', mode='exec')
KeyboardInterrupt
Traceback (most recent call last):
  File "./main_bytes.py", line 4, in <module>
    import numpy
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/__init__.py", line 145, in <module>
    from . import lib
  File "<frozen importlib._bootstrap>", line 202, in _lock_unlock_module
  File "<frozen importlib._bootstrap>", line 78, in acquire
KeyboardInterrupt
Traceback (most recent call last):
  File "./main_bytes.py", line 4, in <module>
    import numpy
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/__init__.py", line 142, in <module>
    from . import core
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/core/__init__.py", line 104, in <module>
    from . import _add_newdocs
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/core/_add_newdocs.py", line 6734, in <module>
    ('complex256', 'Complex number type composed of 2 128-bit extended-precision floating-point numbers'),
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/core/_add_newdocs.py", line 6712, in numeric_type_aliases
    return list(type_aliases_gen())
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/core/_add_newdocs.py", line 6709, in type_aliases_gen
    pass
KeyboardInterrupt
Traceback (most recent call last):
  File "./main_bytes.py", line 34, in <module>
    arr = load_file(fname)
  File "/home/divesh/Desktop/thesis/offline/load_file.py", line 32, in load_file
    cap.apply_on_packets(new_packet)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/capture/capture.py", line 277, in apply_on_packets
    return self.eventloop.run_until_complete(coro)
  File "/usr/lib/python3.6/asyncio/base_events.py", line 471, in run_until_complete
    self.run_forever()
  File "/usr/lib/python3.6/asyncio/base_events.py", line 438, in run_forever
    self._run_once()
  File "/usr/lib/python3.6/asyncio/base_events.py", line 1415, in _run_once
    event_list = self._selector.select(timeout)
  File "/usr/lib/python3.6/selectors.py", line 445, in select
    fd_event_list = self._epoll.poll(timeout, max_ev)
KeyboardInterrupt
Traceback (most recent call last):
  File "./main_bytes.py", line 34, in <module>
    arr = load_file(fname)
  File "/home/divesh/Desktop/thesis/offline/load_file.py", line 32, in load_file
    cap.apply_on_packets(new_packet)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/capture/capture.py", line 277, in apply_on_packets
    return self.eventloop.run_until_complete(coro)
  File "/usr/lib/python3.6/asyncio/base_events.py", line 471, in run_until_complete
    self.run_forever()
  File "/usr/lib/python3.6/asyncio/base_events.py", line 438, in run_forever
    self._run_once()
  File "/usr/lib/python3.6/asyncio/base_events.py", line 1451, in _run_once
    handle._run()
  File "/usr/lib/python3.6/asyncio/events.py", line 145, in _run
    self._callback(*self._args)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/capture/capture.py", line 288, in packets_from_tshark
    await self._go_through_packets_from_fd(tshark_process.stdout, packet_callback, packet_count=packet_count)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/capture/capture.py", line 306, in _go_through_packets_from_fd
    psml_structure=psml_struct)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/capture/capture.py", line 363, in _get_packet_from_stream
    packet = packet_from_xml_packet(packet, psml_structure=psml_structure)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/tshark/tshark_xml.py", line 26, in packet_from_xml_packet
    xml_pkt = lxml.objectify.fromstring(xml_pkt, parser)
KeyboardInterrupt
Failed to import the site module
Traceback (most recent call last):
  File "/usr/lib/python3.6/site.py", line 570, in <module>
    main()
  File "/usr/lib/python3.6/site.py", line 553, in main
    known_paths = venv(known_paths)
  File "/usr/lib/python3.6/site.py", line 490, in venv
    addsitepackages(known_paths, [sys.prefix])
  File "/usr/lib/python3.6/site.py", line 345, in addsitepackages
    addsitedir(sitedir, known_paths)
  File "/usr/lib/python3.6/site.py", line 213, in addsitedir
    addpackage(sitedir, name, known_paths)
  File "/usr/lib/python3.6/site.py", line 174, in addpackage
    exec(line)
  File "<string>", line 1, in <module>
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/_virtualenv.py", line 42, in <module>
    from threading import Lock
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 674, in exec_module
  File "<frozen importlib._bootstrap_external>", line 779, in get_code
  File "<frozen importlib._bootstrap_external>", line 487, in _compile_bytecode
KeyboardInterrupt
Traceback (most recent call last):
  File "./main_bytes.py", line 4, in <module>
    import numpy
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/__init__.py", line 145, in <module>
    from . import lib
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/lib/__init__.py", line 22, in <module>
    from . import mixins
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 951, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 894, in _find_spec
  File "<frozen importlib._bootstrap_external>", line 1157, in find_spec
KeyboardInterrupt
Traceback (most recent call last):
  File "./main_bytes.py", line 4, in <module>
    import numpy
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/__init__.py", line 145, in <module>
    from . import lib
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/lib/__init__.py", line 27, in <module>
    from .index_tricks import *
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/lib/index_tricks.py", line 14, in <module>
    from .function_base import diff
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/lib/function_base.py", line 786, in <module>
    @array_function_dispatch(_gradient_dispatcher)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/core/overrides.py", line 168, in decorator
    verify_matching_signatures(implementation, dispatcher)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/core/overrides.py", line 75, in verify_matching_signatures
    dispatcher_spec = ArgSpec(*getargspec(dispatcher))
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/compat/_inspect.py", line 112, in getargspec
    args, varargs, varkw = getargs(func.__code__)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/compat/_inspect.py", line 76, in getargs
    if not iscode(co):
KeyboardInterrupt
Traceback (most recent call last):
  File "./main_bytes.py", line 4, in <module>
    import numpy
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/__init__.py", line 145, in <module>
    from . import lib
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/lib/__init__.py", line 27, in <module>
    from .index_tricks import *
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/lib/index_tricks.py", line 14, in <module>
    from .function_base import diff
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/lib/function_base.py", line 641, in <module>
    def select(condlist, choicelist, default=0):
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/core/overrides.py", line 181, in decorator
    source, filename='<__array_function__ internals>', mode='exec')
KeyboardInterrupt
Traceback (most recent call last):
  File "./main_bytes.py", line 4, in <module>
    import numpy
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/__init__.py", line 142, in <module>
    from . import core
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/core/__init__.py", line 107, in <module>
    from . import _internal
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/core/_internal.py", line 11, in <module>
    import platform
  File "/usr/lib/python3.6/platform.py", line 174, in <module>
    br'(libc(_\w+)?\.so(?:\.(\d[0-9.]*))?)', re.ASCII)
  File "/usr/lib/python3.6/re.py", line 233, in compile
    return _compile(pattern, flags)
  File "/usr/lib/python3.6/re.py", line 301, in _compile
    p = sre_compile.compile(pattern, flags)
  File "/usr/lib/python3.6/sre_compile.py", line 557, in compile
    def compile(p, flags=0):
KeyboardInterrupt
Traceback (most recent call last):
  File "./main_bytes.py", line 6, in <module>
    import matplotlib.pyplot as plt
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/matplotlib/pyplot.py", line 2282, in <module>
    switch_backend(rcParams["backend"])
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/matplotlib/__init__.py", line 813, in __getitem__
    plt.switch_backend(rcsetup._auto_backend_sentinel)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/matplotlib/pyplot.py", line 204, in switch_backend
    switch_backend(candidate)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/matplotlib/pyplot.py", line 221, in switch_backend
    backend_mod = importlib.import_module(backend_name)
  File "/usr/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/matplotlib/backends/backend_tkagg.py", line 1, in <module>
    from . import _backend_tk
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/matplotlib/backends/_backend_tk.py", line 6, in <module>
    import tkinter as tk
  File "/usr/lib/python3.6/tkinter/__init__.py", line 36, in <module>
    import _tkinter # If this fails your Python may not be configured for Tk
KeyboardInterrupt
Traceback (most recent call last):
  File "./main_bytes.py", line 34, in <module>
    arr = load_file(fname)
  File "/home/divesh/Desktop/thesis/offline/load_file.py", line 32, in load_file
    cap.apply_on_packets(new_packet)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/capture/capture.py", line 277, in apply_on_packets
    return self.eventloop.run_until_complete(coro)
  File "/usr/lib/python3.6/asyncio/base_events.py", line 471, in run_until_complete
    self.run_forever()
  File "/usr/lib/python3.6/asyncio/base_events.py", line 438, in run_forever
    self._run_once()
  File "/usr/lib/python3.6/asyncio/base_events.py", line 1451, in _run_once
    handle._run()
  File "/usr/lib/python3.6/asyncio/events.py", line 145, in _run
    self._callback(*self._args)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/capture/capture.py", line 288, in packets_from_tshark
    await self._go_through_packets_from_fd(tshark_process.stdout, packet_callback, packet_count=packet_count)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/capture/capture.py", line 306, in _go_through_packets_from_fd
    psml_structure=psml_struct)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/capture/capture.py", line 363, in _get_packet_from_stream
    packet = packet_from_xml_packet(packet, psml_structure=psml_structure)
KeyboardInterrupt
Traceback (most recent call last):
  File "./main_bytes.py", line 114, in <module>
    calc(tmin,t,p)
  File "./main_bytes.py", line 50, in calc
    x1,y1 = map(list,sampling_scheme1(x,y,bytes_poller,utilization_poller,tmin,tmax,param))
TypeError: curvature() takes 4 positional arguments but 7 were given
Traceback (most recent call last):
  File "./main_bytes.py", line 6, in <module>
    import matplotlib.pyplot as plt
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/matplotlib/pyplot.py", line 40, in <module>
    from matplotlib.figure import Figure, figaspect
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/matplotlib/figure.py", line 20, in <module>
    from matplotlib import backends, docstring, projections
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/matplotlib/projections/__init__.py", line 4, in <module>
    from mpl_toolkits.mplot3d import Axes3D
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/mpl_toolkits/mplot3d/__init__.py", line 1, in <module>
    from .axes3d import Axes3D
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/mpl_toolkits/mplot3d/axes3d.py", line 33, in <module>
    from . import axis3d
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/mpl_toolkits/mplot3d/axis3d.py", line 435, in <module>
    class ZAxis(Axis):
KeyboardInterrupt
Traceback (most recent call last):
  File "./main_bytes.py", line 4, in <module>
    import numpy
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/__init__.py", line 145, in <module>
    from . import lib
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/lib/__init__.py", line 39, in <module>
    from .npyio import *
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/lib/npyio.py", line 15, in <module>
    from ._datasource import DataSource
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/lib/_datasource.py", line 42, in <module>
    import shutil
  File "/usr/lib/python3.6/shutil.py", line 15, in <module>
    import zlib
KeyboardInterrupt
Traceback (most recent call last):
  File "./main_bytes.py", line 4, in <module>
    import numpy
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/__init__.py", line 145, in <module>
    from . import lib
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/lib/__init__.py", line 29, in <module>
    from .nanfunctions import *
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/lib/nanfunctions.py", line 798, in <module>
    def nancumprod(a, axis=None, dtype=None, out=None):
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/core/overrides.py", line 168, in decorator
    verify_matching_signatures(implementation, dispatcher)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/core/overrides.py", line 74, in verify_matching_signatures
    implementation_spec = ArgSpec(*getargspec(implementation))
  File "<string>", line 14, in __new__
KeyboardInterrupt
Traceback (most recent call last):
  File "./main_bytes.py", line 4, in <module>
    import numpy
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/__init__.py", line 142, in <module>
    from . import core
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/core/__init__.py", line 107, in <module>
    from . import _internal
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/core/_internal.py", line 11, in <module>
    import platform
  File "/usr/lib/python3.6/platform.py", line 116, in <module>
    import sys, os, re, subprocess
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 674, in exec_module
  File "<frozen importlib._bootstrap_external>", line 779, in get_code
  File "<frozen importlib._bootstrap_external>", line 487, in _compile_bytecode
KeyboardInterrupt
Traceback (most recent call last):
  File "./main_bytes.py", line 34, in <module>
    arr = load_file(fname)
  File "/home/divesh/Desktop/thesis/offline/load_file.py", line 32, in load_file
    cap.apply_on_packets(new_packet)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/capture/capture.py", line 277, in apply_on_packets
    return self.eventloop.run_until_complete(coro)
  File "/usr/lib/python3.6/asyncio/base_events.py", line 471, in run_until_complete
    self.run_forever()
  File "/usr/lib/python3.6/asyncio/base_events.py", line 438, in run_forever
    self._run_once()
  File "/usr/lib/python3.6/asyncio/base_events.py", line 1415, in _run_once
    event_list = self._selector.select(timeout)
  File "/usr/lib/python3.6/selectors.py", line 445, in select
    fd_event_list = self._epoll.poll(timeout, max_ev)
KeyboardInterrupt
Traceback (most recent call last):
  File "./main_bytes.py", line 34, in <module>
    arr = load_file(fname)
  File "/home/divesh/Desktop/thesis/offline/load_file.py", line 32, in load_file
    cap.apply_on_packets(new_packet)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/capture/capture.py", line 277, in apply_on_packets
    return self.eventloop.run_until_complete(coro)
  File "/usr/lib/python3.6/asyncio/base_events.py", line 471, in run_until_complete
    self.run_forever()
  File "/usr/lib/python3.6/asyncio/base_events.py", line 438, in run_forever
    self._run_once()
  File "/usr/lib/python3.6/asyncio/base_events.py", line 1451, in _run_once
    handle._run()
  File "/usr/lib/python3.6/asyncio/events.py", line 145, in _run
    self._callback(*self._args)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/capture/capture.py", line 288, in packets_from_tshark
    await self._go_through_packets_from_fd(tshark_process.stdout, packet_callback, packet_count=packet_count)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/capture/capture.py", line 306, in _go_through_packets_from_fd
    psml_structure=psml_struct)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/capture/capture.py", line 363, in _get_packet_from_stream
    packet = packet_from_xml_packet(packet, psml_structure=psml_structure)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/tshark/tshark_xml.py", line 29, in packet_from_xml_packet
    return _packet_from_pdml_packet(xml_pkt)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/tshark/tshark_xml.py", line 37, in _packet_from_pdml_packet
    layers = [Layer(proto) for proto in pdml_packet.proto]
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/tshark/tshark_xml.py", line 37, in <listcomp>
    layers = [Layer(proto) for proto in pdml_packet.proto]
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/packet/layer.py", line 25, in __init__
    attributes = dict(field.attrib)
KeyboardInterrupt
Traceback (most recent call last):
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/core/__init__.py", line 24, in <module>
    from . import multiarray
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/core/multiarray.py", line 14, in <module>
    from . import overrides
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/core/overrides.py", line 7, in <module>
    from numpy.core._multiarray_umath import (
ImportError: PyCapsule_Import could not import module "datetime"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "./main_bytes.py", line 4, in <module>
    import numpy
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/__init__.py", line 142, in <module>
    from . import core
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/core/__init__.py", line 54, in <module>
    raise ImportError(msg)
ImportError: 

IMPORTANT: PLEASE READ THIS FOR ADVICE ON HOW TO SOLVE THIS ISSUE!

Importing the numpy c-extensions failed.
- Try uninstalling and reinstalling numpy.
- If you have already done that, then:
  1. Check that you expected to use Python3.6 from "/home/divesh/Desktop/thesis/venv/bin/python3",
     and that you have no directories in your PATH or PYTHONPATH that can
     interfere with the Python and numpy version "1.18.2" you're trying to use.
  2. If (1) looks fine, you can open a new issue at
     https://github.com/numpy/numpy/issues.  Please include details on:
     - how you installed Python
     - how you installed numpy
     - your operating system
     - whether or not you have multiple versions of Python installed
     - if you built from source, your compiler versions and ideally a build log

- If you're working with a numpy git repository, try `git clean -xdf`
  (removes all files not under version control) and rebuild numpy.

Note: this error has many possible causes, so please don't comment on
an existing issue about this - open a new one instead.

Original error was: PyCapsule_Import could not import module "datetime"

Traceback (most recent call last):
  File "./main_bytes.py", line 4, in <module>
    import numpy
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/__init__.py", line 152, in <module>
    from . import random
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/random/__init__.py", line 181, in <module>
    from . import _pickle
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/random/_pickle.py", line 1, in <module>
    from .mtrand import RandomState
  File "_bit_generator.pxd", line 14, in init numpy.random.mtrand
  File "_bit_generator.pyx", line 40, in init numpy.random._bit_generator
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 951, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 894, in _find_spec
KeyboardInterrupt
Traceback (most recent call last):
  File "./main_bytes.py", line 6, in <module>
    import matplotlib.pyplot as plt
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/matplotlib/__init__.py", line 138, in <module>
    from . import cbook, rcsetup
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/matplotlib/rcsetup.py", line 27, in <module>
    from matplotlib.fontconfig_pattern import parse_fontconfig_pattern
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/matplotlib/fontconfig_pattern.py", line 18, in <module>
    from pyparsing import (Literal, ZeroOrMore, Optional, Regex, StringEnd,
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyparsing.py", line 6381, in <module>
    class pyparsing_common:
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyparsing.py", line 6564, in pyparsing_common
    fnumber = Regex(r'[+-]?\d+\.?\d*([eE][+-]?\d+)?').setName("fnumber").setParseAction(convertToFloat)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyparsing.py", line 3295, in __init__
    self.re = re.compile(self.pattern, self.flags)
  File "/usr/lib/python3.6/re.py", line 233, in compile
    return _compile(pattern, flags)
  File "/usr/lib/python3.6/re.py", line 301, in _compile
    p = sre_compile.compile(pattern, flags)
  File "/usr/lib/python3.6/sre_compile.py", line 562, in compile
    p = sre_parse.parse(p, flags)
  File "/usr/lib/python3.6/sre_parse.py", line 855, in parse
    p = _parse_sub(source, pattern, flags & SRE_FLAG_VERBOSE, 0)
  File "/usr/lib/python3.6/sre_parse.py", line 416, in _parse_sub
    not nested and not items))
  File "/usr/lib/python3.6/sre_parse.py", line 765, in _parse
    p = _parse_sub(source, state, sub_verbose, nested + 1)
  File "/usr/lib/python3.6/sre_parse.py", line 416, in _parse_sub
    not nested and not items))
  File "/usr/lib/python3.6/sre_parse.py", line 488, in _parse
    sourceget()
KeyboardInterrupt
Traceback (most recent call last):
  File "./main_bytes.py", line 4, in <module>
    import numpy
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/__init__.py", line 152, in <module>
    from . import random
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/random/__init__.py", line 181, in <module>
    from . import _pickle
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/random/_pickle.py", line 1, in <module>
    from .mtrand import RandomState
  File "<frozen importlib._bootstrap>", line 968, in _find_and_load
  File "<frozen importlib._bootstrap>", line 148, in __enter__
  File "<frozen importlib._bootstrap>", line 163, in _get_module_lock
KeyboardInterrupt
Traceback (most recent call last):
  File "./main_bytes.py", line 34, in <module>
    arr = load_file(fname)
  File "/home/divesh/Desktop/thesis/offline/load_file.py", line 32, in load_file
    cap.apply_on_packets(new_packet)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/capture/capture.py", line 277, in apply_on_packets
    return self.eventloop.run_until_complete(coro)
  File "/usr/lib/python3.6/asyncio/base_events.py", line 471, in run_until_complete
    self.run_forever()
  File "/usr/lib/python3.6/asyncio/base_events.py", line 438, in run_forever
    self._run_once()
  File "/usr/lib/python3.6/asyncio/base_events.py", line 1435, in _run_once
    for i in range(ntodo):
KeyboardInterrupt
Traceback (most recent call last):
  File "./main_bytes.py", line 34, in <module>
    arr = load_file(fname)
  File "/home/divesh/Desktop/thesis/offline/load_file.py", line 32, in load_file
    cap.apply_on_packets(new_packet)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/capture/capture.py", line 277, in apply_on_packets
    return self.eventloop.run_until_complete(coro)
  File "/usr/lib/python3.6/asyncio/base_events.py", line 471, in run_until_complete
    self.run_forever()
  File "/usr/lib/python3.6/asyncio/base_events.py", line 438, in run_forever
    self._run_once()
  File "/usr/lib/python3.6/asyncio/base_events.py", line 1451, in _run_once
    handle._run()
  File "/usr/lib/python3.6/asyncio/events.py", line 145, in _run
    self._callback(*self._args)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/capture/capture.py", line 288, in packets_from_tshark
    await self._go_through_packets_from_fd(tshark_process.stdout, packet_callback, packet_count=packet_count)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/capture/capture.py", line 306, in _go_through_packets_from_fd
    psml_structure=psml_struct)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/capture/capture.py", line 363, in _get_packet_from_stream
    packet = packet_from_xml_packet(packet, psml_structure=psml_structure)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/tshark/tshark_xml.py", line 29, in packet_from_xml_packet
    return _packet_from_pdml_packet(xml_pkt)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/tshark/tshark_xml.py", line 37, in _packet_from_pdml_packet
    layers = [Layer(proto) for proto in pdml_packet.proto]
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/tshark/tshark_xml.py", line 37, in <listcomp>
    layers = [Layer(proto) for proto in pdml_packet.proto]
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/packet/layer.py", line 31, in __init__
    self._all_fields[attributes['name']] = LayerFieldsContainer(field_obj)
KeyboardInterrupt
Traceback (most recent call last):
  File "./main_bytes.py", line 34, in <module>
    arr = load_file(fname)
  File "/home/divesh/Desktop/thesis/offline/load_file.py", line 32, in load_file
    cap.apply_on_packets(new_packet)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/capture/capture.py", line 277, in apply_on_packets
    return self.eventloop.run_until_complete(coro)
  File "/usr/lib/python3.6/asyncio/base_events.py", line 471, in run_until_complete
    self.run_forever()
  File "/usr/lib/python3.6/asyncio/base_events.py", line 438, in run_forever
    self._run_once()
  File "/usr/lib/python3.6/asyncio/base_events.py", line 1415, in _run_once
    event_list = self._selector.select(timeout)
  File "/usr/lib/python3.6/selectors.py", line 445, in select
    fd_event_list = self._epoll.poll(timeout, max_ev)
KeyboardInterrupt
Traceback (most recent call last):
  File "./main_bytes.py", line 4, in <module>
    import numpy
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/__init__.py", line 142, in <module>
    from . import core
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/core/__init__.py", line 78, in <module>
    from . import numeric
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/core/numeric.py", line 29, in <module>
    from . import shape_base
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/core/shape_base.py", line 14, in <module>
    from . import fromnumeric as _from_nx
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/core/fromnumeric.py", line 18, in <module>
    from . import _methods
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/core/_methods.py", line 14, in <module>
    from numpy.core import _exceptions
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/core/_exceptions.py", line 57, in <module>
    class _UFuncNoLoopError(UFuncTypeError):
KeyboardInterrupt
Traceback (most recent call last):
  File "./main_bytes.py", line 4, in <module>
    import numpy
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/__init__.py", line 145, in <module>
    from . import lib
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/lib/__init__.py", line 39, in <module>
    from .npyio import *
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/lib/npyio.py", line 15, in <module>
    from ._datasource import DataSource
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/lib/_datasource.py", line 42, in <module>
    import shutil
  File "/usr/lib/python3.6/shutil.py", line 15, in <module>
    import zlib
KeyboardInterrupt
Traceback (most recent call last):
  File "./main_bytes.py", line 4, in <module>
    import numpy
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/__init__.py", line 142, in <module>
    from . import core
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/core/__init__.py", line 106, in <module>
    from . import _dtype_ctypes
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/core/_dtype_ctypes.py", line 26, in <module>
    import ctypes
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 674, in exec_module
  File "<frozen importlib._bootstrap_external>", line 779, in get_code
  File "<frozen importlib._bootstrap_external>", line 487, in _compile_bytecode
KeyboardInterrupt
Traceback (most recent call last):
  File "./main_bytes.py", line 4, in <module>
    import numpy
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/__init__.py", line 142, in <module>
    from . import core
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/core/__init__.py", line 107, in <module>
    from . import _internal
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/core/_internal.py", line 11, in <module>
    import platform
  File "/usr/lib/python3.6/platform.py", line 174, in <module>
    br'(libc(_\w+)?\.so(?:\.(\d[0-9.]*))?)', re.ASCII)
  File "/usr/lib/python3.6/re.py", line 233, in compile
    return _compile(pattern, flags)
  File "/usr/lib/python3.6/re.py", line 301, in _compile
    p = sre_compile.compile(pattern, flags)
  File "/usr/lib/python3.6/sre_compile.py", line 562, in compile
    p = sre_parse.parse(p, flags)
  File "/usr/lib/python3.6/sre_parse.py", line 855, in parse
    p = _parse_sub(source, pattern, flags & SRE_FLAG_VERBOSE, 0)
  File "/usr/lib/python3.6/sre_parse.py", line 417, in _parse_sub
    if not sourcematch("|"):
  File "/usr/lib/python3.6/sre_parse.py", line 250, in match
    self.__next()
  File "/usr/lib/python3.6/sre_parse.py", line 234, in __next
    try:
KeyboardInterrupt
Traceback (most recent call last):
  File "./main_bytes.py", line 4, in <module>
    import numpy
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/__init__.py", line 142, in <module>
    from . import core
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/core/__init__.py", line 107, in <module>
    from . import _internal
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/core/_internal.py", line 11, in <module>
    import platform
  File "/usr/lib/python3.6/platform.py", line 116, in <module>
    import sys, os, re, subprocess
  File "/usr/lib/python3.6/subprocess.py", line 137, in <module>
    import select
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 951, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 894, in _find_spec
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/_virtualenv.py", line 51, in find_spec
    def find_spec(self, fullname, path, target=None):
KeyboardInterrupt
Traceback (most recent call last):
  File "./main_bytes.py", line 4, in <module>
    import numpy
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/__init__.py", line 142, in <module>
    from . import core
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/core/__init__.py", line 104, in <module>
    from . import _add_newdocs
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/core/_add_newdocs.py", line 2342, in <module>
    """))
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/core/function_base.py", line 506, in add_newdoc
    new = getattr(__import__(place, globals(), {}, [obj]), obj)
  File "<frozen importlib._bootstrap>", line 1007, in _handle_fromlist
KeyboardInterrupt
Traceback (most recent call last):
  File "./main_bytes.py", line 4, in <module>
    import numpy
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/__init__.py", line 142, in <module>
    from . import core
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/core/__init__.py", line 107, in <module>
    from . import _internal
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/core/_internal.py", line 11, in <module>
    import platform
  File "/usr/lib/python3.6/platform.py", line 116, in <module>
    import sys, os, re, subprocess
  File "/usr/lib/python3.6/subprocess.py", line 50, in <module>
    import signal
  File "/usr/lib/python3.6/signal.py", line 10, in <module>
    lambda name:
  File "/usr/lib/python3.6/enum.py", line 634, in _convert
    cls = cls(name, members, module=module)
  File "/usr/lib/python3.6/enum.py", line 295, in __call__
    return cls._create_(value, names, module=module, qualname=qualname, type=type, start=start)
  File "/usr/lib/python3.6/enum.py", line 401, in _create_
    enum_class = metacls.__new__(metacls, class_name, bases, classdict)
  File "/usr/lib/python3.6/enum.py", line 250, in __new__
    enum_class.__new__ = Enum.__new__
  File "/usr/lib/python3.6/enum.py", line 364, in __setattr__
    super().__setattr__(name, value)
KeyboardInterrupt
Traceback (most recent call last):
  File "./main_bytes.py", line 34, in <module>
    arr = load_file(fname)
  File "/home/divesh/Desktop/thesis/offline/load_file.py", line 32, in load_file
    cap.apply_on_packets(new_packet)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/capture/capture.py", line 277, in apply_on_packets
    return self.eventloop.run_until_complete(coro)
  File "/usr/lib/python3.6/asyncio/base_events.py", line 471, in run_until_complete
    self.run_forever()
  File "/usr/lib/python3.6/asyncio/base_events.py", line 438, in run_forever
    self._run_once()
  File "/usr/lib/python3.6/asyncio/base_events.py", line 1415, in _run_once
    event_list = self._selector.select(timeout)
  File "/usr/lib/python3.6/selectors.py", line 445, in select
    fd_event_list = self._epoll.poll(timeout, max_ev)
KeyboardInterrupt
Traceback (most recent call last):
  File "./main_bytes.py", line 35, in <module>
    arr = load_file(fname)
  File "/home/divesh/Desktop/thesis/offline/load_file.py", line 32, in load_file
    cap.apply_on_packets(new_packet)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/capture/capture.py", line 277, in apply_on_packets
    return self.eventloop.run_until_complete(coro)
  File "/usr/lib/python3.6/asyncio/base_events.py", line 471, in run_until_complete
    self.run_forever()
  File "/usr/lib/python3.6/asyncio/base_events.py", line 438, in run_forever
    self._run_once()
  File "/usr/lib/python3.6/asyncio/base_events.py", line 1451, in _run_once
    handle._run()
  File "/usr/lib/python3.6/asyncio/events.py", line 145, in _run
    self._callback(*self._args)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/capture/capture.py", line 288, in packets_from_tshark
    await self._go_through_packets_from_fd(tshark_process.stdout, packet_callback, packet_count=packet_count)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/capture/capture.py", line 306, in _go_through_packets_from_fd
    psml_structure=psml_struct)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/capture/capture.py", line 363, in _get_packet_from_stream
    packet = packet_from_xml_packet(packet, psml_structure=psml_structure)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/tshark/tshark_xml.py", line 29, in packet_from_xml_packet
    return _packet_from_pdml_packet(xml_pkt)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/tshark/tshark_xml.py", line 37, in _packet_from_pdml_packet
    layers = [Layer(proto) for proto in pdml_packet.proto]
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/tshark/tshark_xml.py", line 37, in <listcomp>
    layers = [Layer(proto) for proto in pdml_packet.proto]
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/packet/layer.py", line 26, in __init__
    field_obj = LayerField(**attributes)
KeyboardInterrupt
Traceback (most recent call last):
  File "./main_bytes.py", line 7, in <module>
    import scipy.interpolate as interpolate
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/scipy/interpolate/__init__.py", line 167, in <module>
    from .interpolate import *
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/scipy/interpolate/interpolate.py", line 15, in <module>
    import scipy.special as spec
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/scipy/special/__init__.py", line 634, in <module>
    from . import _ufuncs
KeyboardInterrupt
Traceback (most recent call last):
  File "./main_bytes.py", line 6, in <module>
    import matplotlib.pyplot as plt
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/matplotlib/pyplot.py", line 32, in <module>
    import matplotlib.colorbar
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/matplotlib/colorbar.py", line 29, in <module>
    import matplotlib.collections as collections
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/matplotlib/collections.py", line 2080, in <module>
    patchstr = artist.kwdoc(Collection)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/matplotlib/artist.py", line 1630, in kwdoc
    'Properties:\n' + '\n'.join(ai.pprint_setters(leadingspace=4)))
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/matplotlib/artist.py", line 1402, in pprint_setters
    accepts = self.get_valid_values(prop)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/matplotlib/artist.py", line 1308, in get_valid_values
    match = re.search(r"(?m)^ *\*?{} : (.+)".format(param_name), docstring)
  File "/usr/lib/python3.6/re.py", line 182, in search
    return _compile(pattern, flags).search(string)
  File "/usr/lib/python3.6/re.py", line 301, in _compile
    p = sre_compile.compile(pattern, flags)
  File "/usr/lib/python3.6/sre_compile.py", line 566, in compile
    code = _code(p, flags)
  File "/usr/lib/python3.6/sre_compile.py", line 548, in _code
    _compile_info(code, p, flags)
  File "/usr/lib/python3.6/sre_compile.py", line 486, in _compile_info
    lo, hi = pattern.getwidth()
  File "/usr/lib/python3.6/sre_parse.py", line 200, in getwidth
    elif op in _UNITCODES:
KeyboardInterrupt
Traceback (most recent call last):
  File "./main_bytes.py", line 4, in <module>
    import numpy
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/__init__.py", line 145, in <module>
    from . import lib
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/lib/__init__.py", line 38, in <module>
    from .arraysetops import *
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 951, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 894, in _find_spec
  File "<frozen importlib._bootstrap_external>", line 1157, in find_spec
  File "<frozen importlib._bootstrap_external>", line 1129, in _get_spec
  File "<frozen importlib._bootstrap_external>", line 1271, in find_spec
  File "<frozen importlib._bootstrap_external>", line 96, in _path_isfile
  File "<frozen importlib._bootstrap_external>", line 88, in _path_is_mode_type
  File "<frozen importlib._bootstrap_external>", line 82, in _path_stat
KeyboardInterrupt
Traceback (most recent call last):
  File "./main_bytes.py", line 4, in <module>
    import numpy
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/__init__.py", line 145, in <module>
    from . import lib
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/lib/__init__.py", line 27, in <module>
    from .index_tricks import *
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/lib/index_tricks.py", line 13, in <module>
    import numpy.matrixlib as matrixlib
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/matrixlib/__init__.py", line 6, in <module>
    from .defmatrix import *
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/matrixlib/defmatrix.py", line 13, in <module>
    from numpy.linalg import matrix_power
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/linalg/__init__.py", line 75, in <module>
    from .linalg import *
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/linalg/linalg.py", line 2317, in <module>
    def norm(x, ord=None, axis=None, keepdims=False):
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/core/overrides.py", line 178, in decorator
    source = _wrapped_func_source.format(name=implementation.__name__)
KeyboardInterrupt
Traceback (most recent call last):
  File "./main_bytes.py", line 4, in <module>
    import numpy
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/__init__.py", line 228, in <module>
    from .testing import Tester
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/testing/__init__.py", line 10, in <module>
    from unittest import TestCase
  File "/usr/lib/python3.6/unittest/__init__.py", line 59, in <module>
    from .case import (TestCase, FunctionTestCase, SkipTest, skip, skipIf,
  File "/usr/lib/python3.6/unittest/case.py", line 6, in <module>
    import logging
  File "/usr/lib/python3.6/logging/__init__.py", line 28, in <module>
    from string import Template
  File "/usr/lib/python3.6/string.py", line 77, in <module>
    class Template(metaclass=_TemplateMetaclass):
  File "/usr/lib/python3.6/string.py", line 74, in __init__
    cls.pattern = _re.compile(pattern, cls.flags | _re.VERBOSE)
  File "/usr/lib/python3.6/re.py", line 233, in compile
    return _compile(pattern, flags)
  File "/usr/lib/python3.6/re.py", line 301, in _compile
    p = sre_compile.compile(pattern, flags)
  File "/usr/lib/python3.6/sre_compile.py", line 562, in compile
    p = sre_parse.parse(p, flags)
  File "/usr/lib/python3.6/sre_parse.py", line 855, in parse
    p = _parse_sub(source, pattern, flags & SRE_FLAG_VERBOSE, 0)
  File "/usr/lib/python3.6/sre_parse.py", line 416, in _parse_sub
    not nested and not items))
  File "/usr/lib/python3.6/sre_parse.py", line 765, in _parse
    p = _parse_sub(source, state, sub_verbose, nested + 1)
  File "/usr/lib/python3.6/sre_parse.py", line 416, in _parse_sub
    not nested and not items))
  File "/usr/lib/python3.6/sre_parse.py", line 765, in _parse
    p = _parse_sub(source, state, sub_verbose, nested + 1)
  File "/usr/lib/python3.6/sre_parse.py", line 416, in _parse_sub
    not nested and not items))
  File "/usr/lib/python3.6/sre_parse.py", line 502, in _parse
    code = _escape(source, this, state)
  File "/usr/lib/python3.6/sre_parse.py", line 344, in _escape
    code = CATEGORIES.get(escape)
KeyboardInterrupt
Traceback (most recent call last):
  File "./main_bytes.py", line 4, in <module>
    import numpy
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/__init__.py", line 228, in <module>
    from .testing import Tester
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/testing/__init__.py", line 10, in <module>
    from unittest import TestCase
  File "/usr/lib/python3.6/unittest/__init__.py", line 58, in <module>
    from .result import TestResult
  File "/usr/lib/python3.6/unittest/result.py", line 7, in <module>
    from . import util
  File "/usr/lib/python3.6/unittest/util.py", line 119, in <module>
    _Mismatch = namedtuple('Mismatch', 'actual expected value')
  File "/usr/lib/python3.6/collections/__init__.py", line 429, in namedtuple
    exec(class_definition, namespace)
  File "<string>", line 1, in <module>
KeyboardInterrupt
Traceback (most recent call last):
  File "./main_bytes.py", line 6, in <module>
    import matplotlib.pyplot as plt
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/matplotlib/pyplot.py", line 32, in <module>
    import matplotlib.colorbar
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/matplotlib/colorbar.py", line 31, in <module>
    import matplotlib.contour as contour
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/matplotlib/contour.py", line 17, in <module>
    import matplotlib.text as text
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/matplotlib/text.py", line 1665, in <module>
    docstring.interpd.update(TextWithDash=artist.kwdoc(TextWithDash))
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/matplotlib/artist.py", line 1630, in kwdoc
    'Properties:\n' + '\n'.join(ai.pprint_setters(leadingspace=4)))
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/matplotlib/artist.py", line 1402, in pprint_setters
    accepts = self.get_valid_values(prop)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/matplotlib/artist.py", line 1292, in get_valid_values
    docstring = inspect.getdoc(func)
  File "/usr/lib/python3.6/inspect.py", line 612, in getdoc
    return cleandoc(doc)
  File "/usr/lib/python3.6/inspect.py", line 628, in cleandoc
    if content:
KeyboardInterrupt
Traceback (most recent call last):
  File "./main_bytes.py", line 35, in <module>
    arr = load_file(fname)
  File "/home/divesh/Desktop/thesis/offline/load_file.py", line 32, in load_file
    cap.apply_on_packets(new_packet)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/capture/capture.py", line 277, in apply_on_packets
    return self.eventloop.run_until_complete(coro)
  File "/usr/lib/python3.6/asyncio/base_events.py", line 471, in run_until_complete
    self.run_forever()
  File "/usr/lib/python3.6/asyncio/base_events.py", line 438, in run_forever
    self._run_once()
  File "/usr/lib/python3.6/asyncio/base_events.py", line 1415, in _run_once
    event_list = self._selector.select(timeout)
  File "/usr/lib/python3.6/selectors.py", line 445, in select
    fd_event_list = self._epoll.poll(timeout, max_ev)
KeyboardInterrupt
Traceback (most recent call last):
  File "./main_bytes.py", line 35, in <module>
    arr = load_file(fname)
  File "/home/divesh/Desktop/thesis/offline/load_file.py", line 32, in load_file
    cap.apply_on_packets(new_packet)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/capture/capture.py", line 277, in apply_on_packets
    return self.eventloop.run_until_complete(coro)
  File "/usr/lib/python3.6/asyncio/base_events.py", line 471, in run_until_complete
    self.run_forever()
  File "/usr/lib/python3.6/asyncio/base_events.py", line 438, in run_forever
    self._run_once()
  File "/usr/lib/python3.6/asyncio/base_events.py", line 1451, in _run_once
    handle._run()
  File "/usr/lib/python3.6/asyncio/events.py", line 145, in _run
    self._callback(*self._args)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/capture/capture.py", line 288, in packets_from_tshark
    await self._go_through_packets_from_fd(tshark_process.stdout, packet_callback, packet_count=packet_count)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/capture/capture.py", line 306, in _go_through_packets_from_fd
    psml_structure=psml_struct)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/capture/capture.py", line 363, in _get_packet_from_stream
    packet = packet_from_xml_packet(packet, psml_structure=psml_structure)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/tshark/tshark_xml.py", line 29, in packet_from_xml_packet
    return _packet_from_pdml_packet(xml_pkt)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/tshark/tshark_xml.py", line 37, in _packet_from_pdml_packet
    layers = [Layer(proto) for proto in pdml_packet.proto]
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/tshark/tshark_xml.py", line 37, in <listcomp>
    layers = [Layer(proto) for proto in pdml_packet.proto]
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/packet/layer.py", line 24, in __init__
    for field in xml_obj.findall('.//field'):
KeyboardInterrupt
Traceback (most recent call last):
  File "./main_bytes.py", line 4, in <module>
    import numpy
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/__init__.py", line 142, in <module>
    from . import core
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/core/__init__.py", line 24, in <module>
    from . import multiarray
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/core/multiarray.py", line 14, in <module>
    from . import overrides
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/numpy/core/overrides.py", line 7, in <module>
    from numpy.core._multiarray_umath import (
KeyboardInterrupt
Traceback (most recent call last):
  File "./main_bytes.py", line 6, in <module>
    import matplotlib.pyplot as plt
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/matplotlib/pyplot.py", line 2282, in <module>
    switch_backend(rcParams["backend"])
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/matplotlib/__init__.py", line 813, in __getitem__
    plt.switch_backend(rcsetup._auto_backend_sentinel)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/matplotlib/pyplot.py", line 204, in switch_backend
    switch_backend(candidate)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/matplotlib/pyplot.py", line 221, in switch_backend
    backend_mod = importlib.import_module(backend_name)
  File "/usr/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/matplotlib/backends/backend_tkagg.py", line 1, in <module>
    from . import _backend_tk
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/matplotlib/backends/_backend_tk.py", line 7, in <module>
    from tkinter.simpledialog import SimpleDialog
  File "/usr/lib/python3.6/tkinter/simpledialog.py", line 27, in <module>
    from tkinter import messagebox
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 951, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 894, in _find_spec
  File "<frozen importlib._bootstrap_external>", line 1157, in find_spec
  File "<frozen importlib._bootstrap_external>", line 1129, in _get_spec
  File "<frozen importlib._bootstrap_external>", line 1271, in find_spec
  File "<frozen importlib._bootstrap_external>", line 96, in _path_isfile
  File "<frozen importlib._bootstrap_external>", line 88, in _path_is_mode_type
  File "<frozen importlib._bootstrap_external>", line 82, in _path_stat
KeyboardInterrupt
Traceback (most recent call last):
  File "./main_bytes.py", line 6, in <module>
    import matplotlib.pyplot as plt
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/matplotlib/pyplot.py", line 40, in <module>
    from matplotlib.figure import Figure, figaspect
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/matplotlib/figure.py", line 20, in <module>
    from matplotlib import backends, docstring, projections
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/matplotlib/projections/__init__.py", line 1, in <module>
    from .. import axes, docstring, cbook
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/matplotlib/axes/__init__.py", line 1, in <module>
    from ._subplots import *
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/matplotlib/axes/_subplots.py", line 6, in <module>
    from matplotlib.axes._axes import Axes
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/matplotlib/axes/_axes.py", line 11, in <module>
    import matplotlib.category as _  # <-registers a category unit converter
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/matplotlib/category.py", line 14, in <module>
    import dateutil.parser
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/dateutil/parser/__init__.py", line 2, in <module>
    from ._parser import parse, parser, parserinfo, ParserError
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/dateutil/parser/_parser.py", line 50, in <module>
    from .. import tz
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/dateutil/tz/__init__.py", line 2, in <module>
    from .tz import *
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/dateutil/tz/tz.py", line 1679, in <module>
    gettz = __get_gettz()
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/dateutil/tz/tz.py", line 1475, in __get_gettz
    class GettzFunc(object):
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/dateutil/tz/tz.py", line 1591, in GettzFunc
    def nocache(name=None):
KeyboardInterrupt
Traceback (most recent call last):
  File "./main_bytes.py", line 35, in <module>
    arr = load_file(fname)
  File "/home/divesh/Desktop/thesis/offline/load_file.py", line 32, in load_file
    cap.apply_on_packets(new_packet)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/capture/capture.py", line 277, in apply_on_packets
    return self.eventloop.run_until_complete(coro)
  File "/usr/lib/python3.6/asyncio/base_events.py", line 471, in run_until_complete
    self.run_forever()
  File "/usr/lib/python3.6/asyncio/base_events.py", line 438, in run_forever
    self._run_once()
  File "/usr/lib/python3.6/asyncio/base_events.py", line 1451, in _run_once
    handle._run()
  File "/usr/lib/python3.6/asyncio/events.py", line 145, in _run
    self._callback(*self._args)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/capture/capture.py", line 286, in packets_from_tshark
    tshark_process = await self._get_tshark_process(packet_count=packet_count)
  File "/home/divesh/Desktop/thesis/venv/lib/python3.6/site-packages/pyshark/capture/capture.py", line 404, in _get_tshark_process
    stdin=stdin)
  File "/usr/lib/python3.6/asyncio/subprocess.py", line 225, in create_subprocess_exec
    stderr=stderr, **kwds)
  File "/usr/lib/python3.6/asyncio/base_events.py", line 1211, in subprocess_exec
    bufsize, **kwargs)
  File "/usr/lib/python3.6/asyncio/unix_events.py", line 200, in _make_subprocess_transport
    **kwargs)
  File "/usr/lib/python3.6/asyncio/base_subprocess.py", line 39, in __init__
    stderr=stderr, bufsize=bufsize, **kwargs)
  File "/usr/lib/python3.6/asyncio/unix_events.py", line 706, in _start
    universal_newlines=False, bufsize=bufsize, **kwargs)
  File "/usr/lib/python3.6/subprocess.py", line 729, in __init__
    restore_signals, start_new_session)
  File "/usr/lib/python3.6/subprocess.py", line 1295, in _execute_child
    restore_signals, start_new_session, preexec_fn)
KeyboardInterrupt
